plutus-ledger-constraints/src/Ledger/Constraints/OffChain.hs

```diff
@@ -569,10 +569,10 @@ adjustUnbalancedTx params = alaf Compose (tx . Tx.outputs . traverse) (adjustTxO
 adjustTxOut :: Params -> TxOut -> Either Tx.ToCardanoError ([Ada.Ada], TxOut)
 adjustTxOut params txOut = do
     -- Increasing the ada amount can also increase the size in bytes, so start with a rough estimated amount of ada
-    withMinAdaValue <- C.toCardanoTxOutValue $ txOutValue txOut <> Ada.toValue minAdaTxOut
+    withMinAdaValue <- C.toCardanoTxOutValue $ txOutValue txOut \/ Ada.toValue (minAdaTxOut params txOut)
```

Comment:
You used `\/` because it's a more general version of `<>`?

Reply:
`\/` is a merge that keeps the max for each currency, so instead of adding the minimum, we set it to the minimum if it isn't reached yet.